pkg/auth/tanzu/kubeconfig.go

```diff
@@ -89,3 +103,25 @@ func getExecConfig(c *configtypes.Context) *clientcmdapi.ExecConfig {
 	execConfig.Args = append([]string{"context", "get-token"}, c.Name)
 	return execConfig
 }
+
+// TanzuLocalKubeConfigPath returns the local tanzu kubeconfig path
+func TanzuLocalKubeConfigPath() (path string, err error) {
```

Comment:
Nit: does this need to be public

Reply:
There was a plan to update the `tanzu` command help to print the tanzu kubeconfig file path, so I made it public. But now with `tanzu context current` this is no longer required. Updated the PR to make it private.We can make later if necessary.